cabot/cabotapp/models.py

```diff
@@ -38,6 +38,16 @@
 )
 
 
+class PositiveIntegerMaxField(models.PositiveIntegerField):
+    def __init__(self, verbose_name=None, name=None, max_value=None, **kwargs):
+        self.max_value = max_value
+        models.PositiveIntegerField.__init__(self, verbose_name, name, **kwargs)
+    def formfield(self, **kwargs):
+        defaults = {'max_value':self.max_value}
+        defaults.update(kwargs)
+        return super(PositiveIntegerMaxField, self).formfield(**defaults)
```

Comment:
nice!! Can we put this in a new file though (fields.py maybe)? The original Cabot devs just shoved everything in models/views but would be good to separate everything out by function in new code